src/Integration/Symfony/Bundle/src/DependencyInjection/AsyncAwsExtension.php

```diff
@@ -137,6 +140,27 @@ private function registerEnvLoader(ContainerBuilder $container, array $config):
                 $config['secrets']['path'],
                 $config['secrets']['recursive'],
             ]);
+
+        if ($config['secrets']['cache']['enabled']) {
+            if (!\interface_exists(CacheInterface::class)) {
+                throw new InvalidConfigurationException(sprintf('You have enabled "async_aws.secrets.cache" but the "symfony/cache" package is not installed. Try running "composer require symfony/cache"'));
+            }
+
+            if (null === $cacheId = $config['secrets']['cache']['pool']) {
```

Comment:
We could set the `pool` to be required. Or we can fallback to cache.system. 

I think this logic could be removed.

Reply:
I hesitate a lot between both solution. This fallback to `cache.system` reduce complexity. 
Fixed thanks